test/services/ArrivalMovementServiceSpec.scala

```diff
@@ -36,14 +38,16 @@ import scala.concurrent.Future
 
 class ArrivalMovementServiceSpec extends SpecBase with IntegrationPatience {
 
+  val dateOfPrep          = LocalDate.now()
+  val timeOfPrep          = LocalTime.of(1, 1)
+  private val arrivalMeta = ArrivalMeta(ArrivalDateTime(dateOfPrep, timeOfPrep), ArrivalDateTime(dateOfPrep, timeOfPrep))
```

Comment:
Can we move these to the only test that it's being used in:     "creates an arrival movement with an internal ref number and a mrn, date and time of creation from the message submitted"